misc/HelloWorldWithWebRoot/src/HelloWorldWithWebRoot/Microsoft.AspNet.Hosting.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Hosting": {
+        "WebRoot": "wwwroot"
```

Comment:
This doesn't just apply to json config files, it could be an environment variable or a command line parameter. Do you really want WebRoot as a environment variable?